scenarios/grpc.js

```diff
@@ -163,3 +163,34 @@ export function uuidv4() {
         return v.toString(16);
     });
 }
+
+export function handleSummary(data) {
+    const neo_ver = (__ENV.NEO_VER || 'Neo_ver_not_set');
+    const test_type = (__ENV.TEST_TYPE || 'Test_type_not_set');
+    const run_id = (__ENV.RUN_ID || 'Run_id_not_set');
+    const test_id = (__ENV.TEST_ID || 'Test_id_not_set');
+    const size_id = (__ENV.SIZE_ID || 'Size_id_not_set');
+    const vu_qty = (__ENV.VU_QTY || 'Vu_qty_not_set');
+    const host_id = (__ENV.HOST_ID || 'Host_id_not_set');
+    const result_format = (__ENV.RESULT_FORMAT || 'not_set');
+    const c1 = ~~((data.metrics.data_received.values.rate || '0')/1024/1024);
+    const c2 = ~~((data.metrics.data_sent.values.rate || '0')/1024/1024);
+    const e3 = (data.metrics.neofs_obj_get_fails && data.metrics.neofs_obj_get_fails.values.rate || '0');
+    const c3 = ~~((data.metrics.neofs_obj_get_total && data.metrics.neofs_obj_get_total.values.rate || '0') - e3);
+    const c4 = ~~(data.metrics.neofs_obj_get_duration && data.metrics.neofs_obj_get_duration.values.avg || '0');
+    const c5 = ~~(data.metrics.neofs_obj_get_duration && data.metrics.neofs_obj_get_duration.values["p(95)"] || '0');
+    const e6 = (data.metrics.neofs_obj_put_fails && data.metrics.neofs_obj_put_fails.values.rate || '0');
+    const c6 = ~~((data.metrics.neofs_obj_put_total && data.metrics.neofs_obj_put_total.values.rate || '0') - e6);
+    const c7 = ~~(data.metrics.neofs_obj_put_duration && data.metrics.neofs_obj_put_duration.values.avg || '0');
+    const c8 = ~~(data.metrics.neofs_obj_put_duration && data.metrics.neofs_obj_put_duration.values["p(95)"] || '0');
+    const e9 = (data.metrics.neofs_obj_delete_fails && data.metrics.neofs_obj_delete_fails.values.rate || '0');
+    const c9 = ~~((data.metrics.neofs_obj_delete_total && data.metrics.neofs_obj_delete_total.values.rate || '0') - e9);
+    const c10 = ~~(data.metrics.neofs_obj_delete_duration && data.metrics.neofs_obj_delete_duration.values.avg || '0');
+    const c11 = ~~(data.metrics.neofs_obj_delete_duration && data.metrics.neofs_obj_delete_duration.values["p(95)"] || '0');
```

Comment:
Do these names mean something special?

Reply:
No, these names are chosen for convenience, they can be changed to something more meaningful.

Reply:
But if I change the names, the output line will be too long.

Reply:
The output line is already too long, but I do not insist